src/Symfony/Component/Validator/Constraints/IsJson.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Imad ZAIRIG <imadzairig@gmail.com>
+ */
+class IsJson extends Constraint
```

Comment:
Maybe `NotJson` ?

Reply:
yes, I think it will to be more telling 👍